MonkeyLoader.Resonite.Integration/Locale/de.json

```diff
@@ -0,0 +1,45 @@
+{
+  "localeCode": "de",
+  "authors": [ "Banane9" ],
+  "messages": {
+    "Settings.MonkeyLoader.Breadcrumb": "Einstellungen",
+
+    "MonkeyLoader.GamePacks.Resonite.MonkeyLoader.MonkeyCount.Value": "{available, plural, =0{Keine Monkeys} other{<b>{active}</b> {active, plural, one{aktiver Monkey} other{aktive Monkeys}} von <b>{available}</b> verfügbaren}}.",
+    "MonkeyLoader.GamePacks.Resonite.MonkeyLoader.EarlyMonkeyCount.Value": "{available, plural, =0{Keine Early Monkeys} other{<b>{active}</b> {active, plural, one{aktiver Early Monkey} other{aktive Early Monkeys}} von <b>{available}</b> verfügbaren}}.",
+    "MonkeyLoader.GamePacks.Resonite.OpenMonkeyLoader.Name": "MonkeyLoader Einstellungen Öffnen",
+    "MonkeyLoader.GamePacks.Resonite.OpenMonkeyLoader.Description": "Öffnet die Einstellungen für den Modloader selbst.",
+
+    "MonkeyLoader.GamePacks.Resonite.Mod.Metadata": "Metadaten",
+    "MonkeyLoader.GamePacks.Resonite.Mod.GroupDescription": "Zeigt eine Übersicht über den Mod.",
```

Comment:
**die** Mod(ifikation)

Reply:
Also in line 13

Reply:
Die Meinungen sind gespalten 😂
https://www.duden.de/rechtschreibung/Mod

Reply:
Have you read the meaning of "Mod, der"?

> Angehöriger einer Gruppe männlicher Jugendlicher, die den Musikstil der 1960er-Jahre und als Kleidung Anzug und Krawatte bevorzugen

Translated:

> Member of a group of male teenagers who prefer the music style of the 1960s and dress in suits and ties

Reply:
Auf wiktionary steht auch für Mod (modifikation) `f, m` - aber ich hab's doch auch schon geändert xP